Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -759,4 +759,8 @@ open class AVFoundationPlayback: Playback {
             view.accessibilityIdentifier = "AVFoundationPlaybackError"
         }
     }
+
+    public func changeSubtitle(style textStyle: [TextStyle]) {
```

Comment:
I think we can put this code near the other subtitle methods. What do you think?

Reply:
Another thing, I think we can increment the unit tests to cover this part of code.
=]